RecoPixelVertexing/PixelTrackFitting/interface/FitUtils.h

```diff
@@ -49,13 +49,11 @@ namespace riemannFit {
   using Vector4f = Eigen::Vector4f;
   using Vector6f = Eigen::Matrix<double, 6, 1>;
 
-  using u_int = unsigned int;
-
   template <class C>
   __host__ __device__ void printIt(C* m, const char* prefix = "") {
 #ifdef RFIT_DEBUG
-    for (u_int r = 0; r < m->rows(); ++r) {
-      for (u_int c = 0; c < m->cols(); ++c) {
+    for (uint r = 0; r < m->rows(); ++r) {
```

Comment:
should not be uint32_t ?

Reply:
I did a minimally intrusive change, but I personaly would use `size_t` for any iteration. I'd love to use a pythonish range like we had in CTA: https://gitlab.cern.ch/cta/CTA/-/blob/master/common/range.hpp, but you can see there are lots of sanity checks which are fine for IO driven applications but not here. Otherwise:
```C++
for (auto i: Range<size_t>(m->rows()) { }
```

Reply:
size_t is 64bits not necessarily the best option (gcc for instance often do not vectorize).
Actually we decided at some point the best option is int32_t (signed) as apparently is the one that compilers optimize best

Reply:
OK, I'll use 'int32_t' then.

Reply:
... hum, it's indeed uint32_t elsewhere in the same file. I don't want to overdo things with a lot of other changes to do, so I will follow your initial advice and make it uint32_t.

Reply:
... and actually, this leads to incompatibility with the template parameter (int), so I will stick to the neutral change initially committed. Let's make the index type optimization a post merge task.